commit 11465f6a7d43f3584fa6ee4a619a25d944c7af8d
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:38:43 2010 -0700

    Staging: brcm80211: wl_cfg80211.c: fix compiler warnings
    
    size_t should be '%zd' in a printf, not '%d'
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
+++ b/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
@@ -1334,7 +1334,7 @@ wl_cfg80211_connect(struct wiphy *wiphy,
 		WL_DBG(("channel (%d), center_req (%d)\n", wl->channel,
 			chan->center_freq));
 	}
-	WL_DBG(("ie (%p), ie_len (%d)\n", sme->ie, sme->ie_len));
+	WL_DBG(("ie (%p), ie_len (%zd)\n", sme->ie, sme->ie_len));
 	err = wl_set_wpa_version(dev, sme);
 	if (unlikely(err))
 		return err;
@@ -4031,7 +4031,7 @@ void *wl_cfg80211_request_fw(s8 *file_na
 		set_bit(WL_FW_LOADING_DONE, &wl->fw->status);
 		fw_entry = wl->fw->fw_entry;
 		if (fw_entry) {
-			WL_DBG(("fw size (%d), data (%p)\n", fw_entry->size,
+			WL_DBG(("fw size (%zd), data (%p)\n", fw_entry->size,
 				fw_entry->data));
 		}
 	} else if (!test_bit(WL_NVRAM_LOADING_DONE, &wl->fw->status)) {
@@ -4044,7 +4044,7 @@ void *wl_cfg80211_request_fw(s8 *file_na
 		set_bit(WL_NVRAM_LOADING_DONE, &wl->fw->status);
 		fw_entry = wl->fw->fw_entry;
 		if (fw_entry) {
-			WL_DBG(("nvram size (%d), data (%p)\n", fw_entry->size,
+			WL_DBG(("nvram size (%zd), data (%p)\n", fw_entry->size,
 				fw_entry->data));
 		}
 	} else {
